apps/webapp/.env.example

```
@@ -40,4 +40,5 @@ VITE_CHATBOT_CF_ACCESS_CLIENT_SECRET=
 VITE_SKIP_CHAT_AUTH_CHECK=
 VITE_POSTHOG_KEY=
 VITE_POSTHOG_HOST=
-VITE_POSTHOG_ENABLED=
\ No newline at end of file
+VITE_POSTHOG_ENABLED=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_POSTHOG_ENABLED key should go before the VITE_POSTHOG_HOST key

(UnorderedKey)